### ISSUE TITLE
Audio Playback issue on Timeline - Using Audio Track

### ISSUE THREAD

**Issue by Vegas10128** [mltframework/shotcut#1419]
When importing mp3 file to audio track it does not start from 0:00. 

Sometimes it will play the audio 10 or 15 seconds into it, 
But the track shows its starting from 0:00

When scrubbing through the audio, the actual audio itself will shift to the starting point of the track, 

Basically the audio itself shifts over to start of track, but continues to do that when editing and or scrubbing. 

Im not sure of any way better how to explain this but i am reporting it as a bug. 
Please take an audio track that you know well. 

Steps taken to create issue. 

Import an mp3 audio file that you know pretty well and how it should sound. 
Place in audio track on timeline, 
Press the play button. 

Determine if the audio track playing has started from the actual beginning or if audio is separate from the timeline track. 

Scrub through the audio track then navigate back to 00:00:00

Press play and see if audio track is starting from the beginning. 

Shotcut Version: 22.12.21

OS/ Windows 10 Home - Version 22H2 - Build 19045.2728
AMD Ryzen 5 2600X
16GB Ram
M.2 Storage

**Comment by ddennedy** [comment]
I reproduced this, and for me it only happens when the mp3 does not have cover art! I see most of my test projects are using MP3 with cover art, and this went unnoticed. Moreover, I tested versions as far back as 21.10.31 and reproduced. So, I am very surprised it was not reported sooner. However, I am glad to report that it was already fixed for the next version for which there is a beta I encourage you to try out https://github.com/mltframework/shotcut/releases/tag/v23.04.03
